app/views/admin/loans/_loans_grid.html.slim

```diff
@@ -53,7 +60,9 @@
     - g.column name: t('activerecord.attributes.loan.first_interest_payment_date'),
       attribute: 'first_interest_payment_date', filter: false, in_html: false
     - g.column name: t('activerecord.attributes.loan.target_end_date'),
-      attribute: 'target_end_date', filter: false, in_html: false
+      attribute: 'target_end_date', in_html: true,
+      detach_with_id: :target_end_date_filter do |loan|
+      - loan.target_end_date ? l(loan.target_end_date) : ''
```

Comment:
Please use the `ldate` helper in application_helper that does the nil check for you.